src/protocol.cpp

```diff
@@ -47,6 +47,8 @@ const char *WTXIDRELAY="wtxidrelay";
 const char *SENDTXRCNCL="sendtxrcncl";
 } // namespace NetMsgType
 
+const std::string NET_MESSAGE_TYPE_OTHER = "*other*";
```

Comment:
This was moved over from `net.cpp`. Should it be in the `NetMsgType` namespace? If so, what about the `allNetMessageTypes` array? There are places in this PR where I have to add +1 to account for the missing 'other' message type, and I'm not sure if it's appropriate for this to be a special case.